src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs

```diff
@@ -5,11 +5,11 @@ public abstract class AuditEntity<TUserId> : Entity, IAuditEntity<TUserId>
 
     public TUserId Creator { get; set; } = default!;
 
-    public DateTime CreationTime { get; set; } = DateTime.Now;
+    public DateTime CreationTime { get; set; } = DateTime.UtcNow;
```

Comment:
Utc seems reasonable, but I probably don't need it. What do I do?

Reply:
Reassign CreationTime or we provide a GetCurrentTime() for easy overloading

Reply:
The current PR resolve it or next?